.github/workflows/browser-tests.yml

```diff
@@ -73,11 +78,16 @@ jobs:
 
             - name: Checkout PR branch
               uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
 
             - name: Setup Job and Install Dependencies
               uses: ./.github/actions/setup-job
 
             - uses: actions/download-artifact@v4
+              with:
+                  pattern: tachometer-results-*
+                  merge-multiple: true
```

Comment:
does merging multiple change the path or what would the output be for this download? I ask because the logic in the script that is ran to generate the comment is based on the directory structure of this download.

Reply:
By setting this to true, it ensures that the downloaded artifacts will be in the same directory specified by path. If multiple matches are found and this is not set to true, it would default to putting them in different locations.

Reply:
For reference: https://github.com/actions/download-artifact

Reply:
thank you for sharing this resoure! nice add